src/toil/jobStores/abstractJobStore.py

```diff
@@ -78,6 +78,16 @@ def __init__(self, url: ParseResult) -> None:
         """
         super().__init__("The URL '%s' is invalid." % url.geturl())
 
+class UnimplementedURLException(RuntimeError):
+    def __init__(self, url: ParseResult, operation: str) -> None:
+        """
+        :param url: The given URL
```

Comment:
Not sure this docstring is needed, but if kept, it's missing the second "operation" param.